src/representation_theory/group_cohomology_resolution.lean

```diff
@@ -233,6 +233,85 @@ module.free.of_basis (of_mul_action_basis k G n)
 
 end basis
 end group_cohomology.resolution
+namespace finsupp
+-- I am not sure where to put these
```

Comment:
My understanding is that these lemmas are now moved to a more appropriate place in a PR which depends on this one! So maybe backport the commits to this PR too?